src/components/Personalization/in-app-message-actions/actions/displayIframeContent.js

```diff
@@ -137,10 +179,30 @@ const createContainerElement = settings => {
   return element;
 };
 
-const displayHTMLContentInIframe = (settings, collect) => {
-  removeElements(ELEMENT_TAG_CLASSNAME);
+export const createOverlayElement = parameter => {
+  const element = document.createElement("div");
+  const backdropOpacity = parameter.backdropOpacity || 0.5;
+  const backdropColor = parameter.backdropColor || "#FFFFFF";
+  element.id = OVERLAY_TAG_ID;
+  element.className = `${OVERLAY_TAG_CLASSNAME}`;
+
+  Object.assign(element.style, {
+    position: "fixed",
+    top: "0",
+    left: "0",
+    width: "100%",
+    height: "100%",
+    zIndex: "1",
+    background: "transparent",
+    opacity: backdropOpacity,
+    backgroundColor: backdropColor
+  });
+
+  return element;
+};
 
-  const { content, contentType } = settings;
+export const displayHTMLContentInIframe = (settings, collect) => {
+  const { content, contentType, mobileParameters } = settings;
 
```

Comment:
call `dismissMessage()` here.  Since any pre-existing message should be removed if one exists. (only a single message and overlay should be displayed at a time)

Reply:
Done